test_data/config.toml

```diff
@@ -296,3 +296,32 @@ post_cmd = '''
     cp .flox/env/manifest.lock "$env_dir/manifest.lock"
     mv "$RESPONSE_FILE" "$env_dir/$(basename $RESPONSE_FILE)"
 '''
+
+# Starts a Python HTTP server, for use with services
+[envs.python_http_server]
+skip_if_output_exists = "envs/python_http_server"
+files = ["envs/python_http_server/manifest.toml"]
+pre_cmd = "flox init"
+cmd = "flox edit -f manifest.toml"
+post_cmd = '''
+    envs_output_dir="$(dirname $RESPONSE_FILE)"
+    env_dir="$envs_output_dir/python_http_server"
+    mkdir -p "$env_dir"
+    cp manifest.toml "$env_dir/manifest.toml"
+    cp .flox/env/manifest.lock "$env_dir/manifest.lock"
+    mv "$RESPONSE_FILE" "$env_dir/$(basename $RESPONSE_FILE)"
+'''
+
+# A trivial service that just creates a file
+[envs.touch_file_service]
+skip_if_output_exists = "envs/touch_file_service"
+files = ["envs/touch_file_service/manifest.toml"]
+pre_cmd = "flox init"
+cmd = "flox edit -f manifest.toml"
+post_cmd = '''
+    envs_output_dir="$(dirname $RESPONSE_FILE)"
+    env_dir="$envs_output_dir/touch_file_service"
+    mkdir -p "$env_dir"
+    cp manifest.toml "$env_dir/manifest.toml"
+    cp .flox/env/manifest.lock "$env_dir/manifest.lock"
+'''
```

Comment:
blocking: afaict this doesn't use the catalog, so we can just commit these files directly rather than using gen_data. I guess there's some merit to regenerating lockfiles, but I don't think that's consistent with the rest of our tests
```suggestion
```

Reply:
Agree, removed.